test/modules/diten/returnClassDiffModule.good

```diff
@@ -1,3 +1 @@
-returnClassDiffModule.chpl:12: In function 'main':
-returnClassDiffModule.chpl:14: error: unresolved call 'unmanaged C.foo()'
-returnClassDiffModule.chpl:14: note: because no functions named foo found in scope
+3
```

Comment:
In .chpl, let us switch from `return new unmanaged C();` to just `return new C();`
for added coolness and elegance.

Ditto returnClassDiffModule6.chpl and testOpaqueClass*